server/datastore/mysql/maintained_apps.go

```diff
@@ -163,3 +163,30 @@ WHERE NOT EXISTS (
 
 	return avail, meta, nil
 }
+
+// GetSoftwareTitleIDByAppID returns the software title ID related to a given fleet library app ID.
+func (ds *Datastore) GetSoftwareTitleIDByAppID(ctx context.Context, appID uint, teamID *uint) (uint, error) {
```

Comment:
Should we rename this to `GetSoftwareTitleIDByMaintainedAppID`? Currently it sounds as if it could work for any VPP app ID (though the appID would be a string, but still). Feel free to ignore if pressed by time.